frontend/packages/component-library/src/formFieldWrapper/formFieldWrapper.module.scss

```
@@ -0,0 +1,119 @@
+@import '@code-dot-org/component-library-styles/mixins.scss';
+@import '@code-dot-org/component-library-styles/colors.scss';
+@import '@code-dot-org/component-library-styles/primitiveColors.scss';
+@import '@code-dot-org/component-library-styles/typography.module.scss';
+
+// Label common styles
+.formFieldWrapper {
+  display: inline-flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 0.125rem;
+  min-width: 18.75rem;
+  margin: 0;
+
+  .formFieldWrapperLabel {
+    display: inline-flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 0.125rem;
+    margin: 0;
+  }
+
+  .formFieldWrapperHelper {
+    display: flex;
+    align-items: baseline;
+    align-self: stretch;
+  }
+}
+
+// Label colors
+.formFieldWrapper.formFieldWrapper-black,
+.formFieldWrapper.formFieldWrapper-gray {
+  .formFieldWrapperLabelText {
+    color: var(--text-neutral-primary);
+  }
+
+  .formFieldWrapperHelper {
+    color: var(--text-neutral-primary);
+  }
+
+  .formFieldWrapperError {
+    color: var(--text-error-primary);
+  }
+
+  // Disabled styles
+  &:has(:disabled) {
+    .formFieldWrapperLabelText {
+      color: var(--text-neutral-disabled);
+    }
+
+    .formFieldWrapperHelper,
+    .formFieldWrapperError {
+      color: var(--text-neutral-disabled);
+    }
+  }
+}
+
+.formFieldWrapper.formFieldWrapper-white {
+  .formFieldWrapperLabelText {
+    color: var(--text-neutral-inverse);
+  }
+
+  .formFieldWrapperHelper,
+  .formFieldWrapperError {
+    color: var(--text-neutral-inverse);
+  }
+
+  // Disabled styles
+  &:has(:disabled) {
+    .formFieldWrapperLabelText {
+      color: var(--text-neutral-tertiary);
+    }
+
+    .formFieldWrapperHelper,
+    .formFieldWrapperError {
+      color: var(--text-neutral-tertiary);
+    }
+  }
+}
+
+// Label sizes
+.formFieldWrapper-l {
+  .formFieldWrapperLabelText {
+    @include label-one;
+    margin-bottom: 0;
+  }
```

This is optional and not blocking, but we've been using [this new pattern](https://github.com/code-dot-org/code-dot-org/blob/95527bb6a862b0aa8eb2f753b98b8d33dba242cc/frontend/packages/component-library/src/cms/section/section.module.scss#L18-L55) for style options like this since it's self-documenting and easier to parse. 

So it would be like:
```suggestion
// Label sizes
.formFieldWrapper {
  &-l {
    .formFieldWrapperLabelText {
      @include label-one;
      margin-bottom: 0;
    }

    .formFieldWrapperHelper {
      @include body-two;
      margin-bottom: 0;
      gap: 0.375rem;
    }
  }

  &-m {
    .formFieldWrapperLabelText {
      @include label-two;
      margin-bottom: 0;
    }

    .formFieldWrapperHelper {
      @include body-three;
      margin-bottom: 0;
      gap: 0.375rem;
    }
...
```
etc…